scripts/mkcloud

```diff
@@ -243,11 +243,27 @@ function onhost_stop_chef_clients_on_nodes
         '
 }
 
+function onhost_start_chef_clients_on_nodes
+{
+    # Reverse effects of onhost_stop_chef_clients_on_nodes
+    ssh $sshopts root@$adminip '
+            set -x
+            for node in $(crowbar machines list | grep ^d) ; do
+            (
+                echo "Starting chef-client on $node"
+                timeout 30 ssh $node rcchef-client start
+            )
+            done
+        '
+}
+
 function error_exit
 {
     exitcode=$1
     message=$2
-    onhost_supportconfig
+    if [[ -n $BUILD_NUMBER ]]; then
```

Comment:
nitpicking: `-n` is superfluous ... but its explicit

Reply:
IMHO it's better to be explicit, e.g.

- Not everyone is familiar with bash shortcuts, and `-n` is POSIX - even though `[[ ]]` isn't, the meaning is obvious
- It's much more easily to `grep`